src/lib/AppRegistry.tsx

```diff
@@ -49,21 +51,23 @@ class RootContainer extends React.Component<Props, {}> {
           this.state.retrying = true
           return <LoadFailureView onRetry={retry} style={{ flex: 1 }} />
         }}
-        {...untypedProps}
       />
     )
   }
 }
 
-class Artist extends RootContainer {
+class Artist extends React.Component<Props, {}> {
   constructor(props) {
     super(props)
     this.component = Containers.Artist
     this.route = new Routes.Artist({ artistID: props.artistID })
   }
```

Comment:
These properties defined in the constructor could also just go inline where the RootContainer is used inside the render method.